CONTRIBUTING.md

```diff
@@ -129,6 +129,9 @@ Before you submit your pull request consider the following guidelines:
     git rebase master -i
     git push origin my-fix-branch -f
     ```
+    
+
+WARNING. Forced push may delete all in-code comments made in your previous PR
```

Comment:
I don't think it disappears, but it will be difficult to correlate with the new commit.